src/EMS/RingdownStatus.js

```diff
@@ -124,12 +135,23 @@ function RingdownStatus({ className, onStatusChange, ringdown }) {
               type="warning"
               title="Cancel Delivery?"
               message="Patient data will be erased and hospital will be notified."
-              cancel="Maintain delivery"
-              destructive="Confirm cancel delivery"
+              cancel="Keep status"
+              destructive="Update status"
```

Comment:
The button labels were correct before...

<img width="281" alt="Screen Shot 2021-07-14 at 7 05 27 PM" src="https://user-images.githubusercontent.com/140766/125716728-5486ab0d-58d6-4399-9c94-e915a0a27db1.png">
?